### ISSUE TITLE
Include links to related cards

### ISSUE THREAD

**Issue by NilsEnevoldsen** [scryfall/feedback#27]
This isn't trivial to implement, but I think it's worth considering.

Sometimes, when viewing a card, you might like to see related cards, such as:
* Cards in the same [cycle](http://mtg.gamepedia.com/Cycle) or [mega cycle](http://mtg.gamepedia.com/Mega_cycle)
* Cards with related effects, such as those [MythicSpoiler includes in their "Thesaurus" section](http://mythicspoiler.com/akh/cards/asforetold.html), including
   * Cards that are better or strictly better
   * Cards that are "fixed" versions of other cards
* Time Spiral homages
* Planar Chaos colorshifts
* Cards that are linked by flavor, such as Dragonlords with their Commands
* Cards with exceptional synergy, like Niv-Mizzet, the Firemind with Curiosity
* Links to popular decklists that run a card

You might want to include an easy method for user suggestions ([as MythicSpoiler does](http://mythicspoiler.com/tsp/cards/magusofthedisk.html)), as it's nearly impossible to find everything.

Some of these you likely think are bad ideas. Implementing any of them would take some thought. At its simplest, this is a suggestion to include links to other cards in the cycle, which is often something I personally want to see.

**Comment by gmac** [comment]
As always, thanks for the feedback! We established a brain trust on this very subject a while back and came up with many similar ideas to your own. As you point out though, many of these relationships are highly conceptual and not clear cut as development objectives. That definitely slows our momentum on this type of work, though do know that we're always thinking about it.

**Comment by NilsEnevoldsen** [comment]
Agreed. If you were to launch this feature, I think you'd most likely start with unambiguous cycles. [MTGSalvation lists most of them](http://mtg.gamepedia.com/Apocalypse#Cycles), but this would still require a lot of elbow grease, and it makes sense to put it on the back burner.
